crates/holochain/src/core/workflow/publish_dht_ops_workflow/publish_query.rs

```diff
@@ -181,10 +189,21 @@ mod tests {
                 .collect::<Vec<_>>(),
             expected
                 .results
-                .into_iter()
+                .iter()
+                .cloned()
                 .map(|op| op.into_inner().1.to_kitsune())
                 .collect::<Vec<_>>(),
         );
+
+        let num_to_publish = db
+            .to_db()
+            .read_async(|txn| num_still_needing_publish(&txn, agent))
+            .await
+            .unwrap();
+
+        // +1 because `get_ops_to_publish` will filter on `last_publish_time` where `num_still_needing_publish` should
```

Comment:
Note this comment, the count query is intentionally ignoring the `last_publish_time` so that part of the query should not match the `get_ops_to_publish` query

Reply:
Thanks, that would have been hard to reverse engineer later!